dev-docs/get-started-with-prebid-server.md

```diff
@@ -128,12 +126,6 @@ If your `s2sConfig.endpoint` points to a url containing the path `/openrtb2/`, s
 **Aliasing Prebid Server only bidders**
 If you wish to set/use an alias for a Prebid Server only bidder, simply list the alias in your `s2sConfig.bidders` field and call the [`pbjs.aliasBidder` method](http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.aliasBidder) in your prebid code (prior to the `pbjs.requestBids`) to register the alias.
 
-{: .alert.alert-info :}
```

Comment:
if the default is false, why do we want to remove the explanation of what happens when the user sets it to true?

Reply:
@jeanstemp -- Figured that the details should be in the Publisher API reference